File: articles/algorithmic-accountability-and-audit-trails/fortran/accountability_capacity_score.f90
program accountability_capacity_score
  implicit none
  real, dimension(6) :: scores
  real :: capacity
  scores = (/0.72, 0.68, 0.64, 0.58, 0.52, 0.66/)
  capacity = sum(scores) / 6.0
  print *, "accountability_capacity_score=", capacity
end program accountability_capacity_score

File: articles/algorithmic-accountability-and-audit-trails/rust/accountability_capacity_score.rs
fn main() {
    let scores = [0.72_f64, 0.68_f64, 0.64_f64, 0.58_f64, 0.52_f64, 0.66_f64];
    let total: f64 = scores.iter().sum();
    let capacity = total / scores.len() as f64;
    println!("accountability_capacity_score={:.4}", capacity);
}
